internal/db/schema/migrations/oss/postgres/29/04_kms_keys.up.sql

```diff
@@ -0,0 +1,213 @@
+
+-- convert database DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'database', create_time
+from kms_database_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, database_key_id, root_key_version_id, version, key, create_time
+from kms_database_key_version;
+
+-- convert oplog DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'oplog', create_time
+from kms_oplog_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, oplog_key_id, root_key_version_id, version, key, create_time
+from kms_oplog_key_version;
+
+-- convert the audit DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'audit', create_time
+from kms_audit_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, audit_key_id, root_key_version_id, version, key, create_time
+from kms_audit_key_version;
+
+-- convert the oidc DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'oidc', create_time
+from kms_oidc_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, oidc_key_id, root_key_version_id, version, key, create_time
+from kms_oidc_key_version;
+
+-- convert the token DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'tokens', create_time
+from kms_token_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, token_key_id, root_key_version_id, version, key, create_time
+from kms_token_key_version;
+
+-- convert the session DEK tables
+insert into kms_data_key (private_id, root_key_id, purpose, create_time)
+select private_id, root_key_id, 'sessions', create_time
+from kms_session_key;
+
+insert into kms_data_key_version (private_id, data_key_id, root_key_version_id, version, key, create_time)
+select private_id, session_key_id, root_key_version_id, version, key, create_time
+from kms_session_key_version;
+
+-- ############################################################################
+-- post conversion, we add the required triggers
+create trigger default_create_time_column
+before
+insert on kms_data_key
+  for each row execute procedure kms_default_create_time();
+
+create trigger default_create_time
+before
+insert on kms_data_key_version
+  for each row execute procedure kms_default_create_time();
+
+create trigger kms_version_column
+before insert on kms_data_key_version
+	for each row execute procedure kms_version_column('data_key_id');
```

Comment:
Why is the `data_key_id` column being updated instead of the `version` column?

Reply:
I don't believe the `data_key_id` isn't being updated... but it is being used to find the correct row when updating the `version` column.